src/lib/simulation/simulation.ts

```diff
@@ -79,12 +80,15 @@ export const simulateEVMTransactionWithTenderly = async (
   blockNumber?: string,
 ): Promise<{ message: string; success: boolean; rawSimulationResult: any }> => {
   const tenderly = new Tenderly();
+  const amount = mintInstructions.supportsQuantity
```

Comment:
Currently it will simulate both 1, and 2 items.

Failure to simulate 2 will fail pack ingestion in Ecomm. We should change that to faillure to simulate 2 falls back to disabling quantity in the future.

Reply:
the desired logic is if supports quantity, sim 2? else sim 1? 👍

Reply:
It actually runs both 1 _and_ 2 for quantity here → 
https://github.com/floornfts/mobile-minting/blob/main/test/shared/basic-ingestor-tests.ts#L62

Reply:
It actually runs both 1 and 2 for quantity here →
https://github.com/floornfts/mobile-minting/blob/main/test/shared/basic-ingestor-tests.ts#L62